docs/index.rst

```diff
@@ -85,8 +84,13 @@ To see other options for the command, run:
 Accessing Services from the Fixtures
 ------------------------------------
 
+<<<<<<< HEAD
 In some cases you may need to access your application's services inside a fixtures
 class. No problem! Your fixtures class is a service, so you can use normal dependency
+=======
```

Comment:
Oups, I missed this conflict when I rebased the branch. Fix: https://github.com/doctrine/DoctrineFixturesBundle/pull/533